beginner/SumEarnings_Araetoro

```diff
@@ -0,0 +1,17 @@
+
+
+def sum_earnings(input):
+    try:
+        num_of_nums = input.count(",") +1
+        input_value = list(map(int, input.split(",")))
+        if len(input_value) != num_of_nums:
+            raise ValueError
+        balance = 0
+        for num in input_value:
+            balance += num
+            if balance < 0:
+                balance = 0
+        print(balance)
```

Comment:
hey nice work for your first go!

when you are working on algorithm solutions its a best practice to have your functions return the result. then to call the function and print what it returns to check your outputs.

try refactoring your function to return the solution